daemon/containerd/image_exporter.go

```diff
@@ -58,18 +60,28 @@ func (i *ImageService) ExportImage(ctx context.Context, names []string, outStrea
 		archive.WithPlatform(platform),
 	}
 
-	ctx, release, err := i.client.WithLease(ctx)
+	contentStore := i.client.ContentStore()
+	leasesManager := i.client.LeasesService()
+	lease, err := leasesManager.Create(ctx, leases.WithRandomID())
 	if err != nil {
-		return errdefs.System(err)
+		return err
```

Comment:
afaict, this change makes it so we're now leaking a c8d error, is that what we want?